ShopsTable.API/Repositories/AssortmentRepository.cs

```diff
@@ -0,0 +1,42 @@
+﻿using ShopsTable.API.Context;
+using ShopsTable.API.Models;
+using ShopsTable.API.Repositories.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShopsTable.API.Repositories
+{
+    public class AssortmentRepository : IAssortmentRepository
+    {
+        private readonly ShopContext _db;
+
+        public AssortmentRepository(ShopContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<Assortment> CreateAsync(Assortment assortment)
+        {
+            await _db.Assortments.AddAsync(assortment);
+            await _db.SaveChangesAsync();
+            return assortment;
+        }
+
+        public Assortment Get(Guid id)
+        {
+
+            return _db.Assortments.FirstOrDefault(g => string.Equals(g.Id, id));
+        }
+        public IEnumerable<Assortment> Get()
+        {
+            return _db.Assortments;
```

Comment:
Add .ToListAsync()
and List<Assortiment> instead of IEnumerable or if you messed .ToListAsync() use IQueryable.